CONTRIBUTING.md

```diff
@@ -30,7 +30,7 @@ If you want to propose code changes for the Dafny project, please note:
 - To propose code changes, use the standard Github multi-user project process, which is described for Dafny on the [wiki](https://github.com/dafny-lang/dafny/wiki/Setting-up-a-development-copy-of-Dafny).
 
 If your change is user-visible, then part of the PR should be corresponding changes to the
-[`RELEASE_NOTES.md`](../RELEASE_NOTES.md) file and to the 
+[`RELEASE_NOTES.md`](./RELEASE_NOTES.md) file and to the 
```

Comment:
```suggestion
[`RELEASE_NOTES.md`](./RELEASE_NOTES.md) file (by following [this](./docs/dev/README.md)) and to the 
```

Reply:
I wouldn't have guessed that there are actually instructions for the files in [docs/dev/news](https://github.com/dafny-lang/dafny/tree/master/docs/dev/news) so yes we should.